karyon3-api/src/main/java/com/netflix/karyon/spi/health/AbstractHealthIndicator.java

```diff
@@ -1,8 +1,11 @@
-package com.netflix.karyon.health;
+package com.netflix.karyon.spi.health;
```

Comment:
Class in the 'api' package are class that are called by the user of karyon.  Classes in 'spi' are classes that are supposed to be implemented.  This distinction can be confusing and sometimes ambiguous but I thought I'd give it a try.